changelog/6240.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Log catching mechanism creates LogCaptureHandler if necessary.
```

Comment:
I would remove this, since IMO this is outdated.

Reply:
Done